modules/opensearch/includes/opensearch.search.inc

```diff
@@ -585,18 +569,18 @@ function opensearch_search_sort_options() {
   if ($default_sort === 'rank_frequency' && empty($custom_ranking)) {
     $options = array_merge($options, [
       'rank_title' => [
-        'label' => t('Best match on title'),
-        'label_short' => t('Title'),
+        'label' => t('Title', array(), array('context' => 'opensearch_ranking')),
```

Comment:
there's no disambiguation on context in Ding2, så a more unique string might be preferable.

Reply:
I cant really make a more unique string, as the context of the translation then wont make sense

But actually, context does work in Ding2 :)
Although it's hardly even used, if you look in the da.po file, it actually does include the context, which t() uses:

https://github.com/ding2/ding2/pull/1173/files#diff-e45508e5f2bf7e01992b1120a4d3e34fR67206

I've tested it locally and it does seem to work as intended :)